### PR TITLE
show intent and test_show added

### PR DESCRIPTION
**Pull request by anmoliiitd823**
Adding show intent which provide functionality for the user to show the data that is required by applying various aspects.

### CODE REVIEW

**Comment by anmoliiitd823** [comment]
@googlebot I fixed it.

File: intents/show.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+	metric = kwargs.get('metric',None)
+
+	dimensions = kwargs.get('dimensions',None)
+
+	table = aspects.apply_date_range(table, date_range,date_column_name, date_format)
+
+	table = aspects.slice_table(table, slices)
+
+    # collecting the colums not to be removed
```
Comment (arijitde-goog) [inline intents/show.py:88]: fix indent

File: intents/test_show.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+2         SS Iyer            0
+3         WP Saha            7"""
+    assert(expected_result == query_result.to_string())
+
+
+print(test_1.__doc__)
+test_1()    
+print(test_2.__doc__)
+test_2()
+
```
Comment (arijitde-goog) [inline intents/test_show.py:82]: Add a few more tests with more input combinations eg without date range.

Comment (anmoliiitd823) [reply intents/test_show.py:82]: Will add them for some more examples

Comment (anmoliiitd823) [reply intents/test_show.py:218]: More test added , with different kind of parameters such as with and without date range , without metric , counting distinct , etc.

File: intents/show.py (hunk 1 above)
Comment (anmoliiitd823) [reply intents/show.py]: Indentation fixed

**Pull request merged**